Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -202,7 +203,14 @@ void PcapLiveDevice::statsThreadMain()
 pcap_t* PcapLiveDevice::doOpen(const DeviceConfiguration& config)
 {
 	char errbuf[PCAP_ERRBUF_SIZE] = {'\0'};
-	pcap_t* pcap = pcap_create(m_Name.c_str(), errbuf);
+	std::string device_name = m_Name;
+
+	if (device_name == NFLOG_IFACE)
+	{
+		device_name += ":" + std::to_string(config.extra & 0xffff);
```

Comment:
What should be the default value of the group name if not provided by the user? is it possible to capture packets from `nflog:0`?

Reply:
By default, if the NFLOG group is not defined, it is 0, so connecting to `nflog` is the same as `nflog:0`